crowbar_framework/app/controllers/installer/upgrades_controller.rb

```diff
@@ -323,9 +323,14 @@ def post_cleanup_repos
           false
         )
 
-        Crowbar::Repository.chef_data_bag_destroy(
-          "#{repository.data_bag_name}/#{repository.id}"
-        )
+        begin
+          Crowbar::Repository.chef_data_bag_destroy(
+            "#{repository.data_bag_name}/#{repository.id}"
+          )
+        rescue Net::HTTPServerException
```

Comment:
Shouldnt we try to catch an specific error code (404 in this case) rather than caching _any_ error code? That could lead to hiding issues behind a generic catch-all